b2share/modules/records/cli.py

```diff
@@ -102,6 +104,49 @@ def check_and_update_handle_records(update, verbose):
                         click.secho('  file PID ok: {}'.format(pid))
 
 
+@b2records.command()
+@with_appcontext
+@click.option('-u', '--update', is_flag=True, default=False)
+@click.argument('record_id', required=True)
+def check_handles(update, record_id):
+    """Allocates handles for a record and its files, if necessary"""
+    rec_pid = RecordUUIDProvider.get(pid_value=record_id).pid
```

Comment:
`record_id` confuses me here. The record id is normally its internal uuid, but here it seems to be the value of the b2rec PID pointing to the record. Those are two different things. If it is the record's b2rec pid value we could name this variable `record_pid` or `b2rec_pid`.